packages/commonwealth/client/scripts/views/components/archive_tray_with_tooltip.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { CWTooltip } from './component_kit/cw_popover/cw_tooltip';
+import moment from 'moment';
+import { CWIcon } from './component_kit/cw_icons/cw_icon';
+
+type ArchiveTrayWithTooltipProps = {
+  archivedAt: moment.Moment;
+};
+
+export const ArchiveTrayWithTooltip = ({
+  archivedAt,
+}: ArchiveTrayWithTooltipProps) => {
+  return (
+    <CWTooltip
+      hasBackground={true}
+      placement="right"
+      content={`Archived on ${archivedAt.format('MM/DD/YYYY')}`}
+      renderTrigger={(handleInteraction) => (
+        <CWIcon
+          iconName="archiveTrayFilled"
+          iconSize="small"
+          onMouseEnter={handleInteraction}
+          onMouseLeave={handleInteraction}
+        />
+      )}
+    />
+  );
+};
```

Comment:
change name of the file archive_tray_with_tooltip.tsx => ArchiveTrayWithTooltip.tsx

Reply:
Done!